server/src/http/mod.rs

```diff
@@ -159,10 +163,20 @@ async fn show_console() -> anyhow::Result<Response<Body>> {
 }
 
 async fn handle_request(
+    authorizer: Arc<Mutex<Box<dyn Authorizer + Send + Sync>>>,
```

Comment:
why do we need a mutex here? we don't seem to need exclusive access anywhere

Reply:
fixed